src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import { FC, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App: React.FC = () => {
+export const App: FC = () => {
+  const [searchQuery, setSearchQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(({ title, description }) => {
+    const normalizeTitle = title.toLowerCase().trim();
+    const normalizeDescription = description.toLowerCase().trim();
+    const normalizeSearchQuery = searchQuery.toLowerCase().trim();
+
+    return normalizeTitle.includes(normalizeSearchQuery)
+      || normalizeDescription.includes(normalizeSearchQuery);
```

Comment:
u can do like that or just create separate variables to make the return line shorter

```suggestion
	const normalizeTitle = title.toLowerCase().trim();
	
	const condition1 = description
	  .toLowerCase()
	  .trim()
	  .includes(normalizeSearchQuery);
	
	const condition2 = searchQuery
	  .toLowerCase()
	  .trim()
	  .includes(normalizeSearchQuery);;
	
	return condition1 || condition2;
```